terraform/cloud-platform-components/README.md

```diff
@@ -111,29 +127,32 @@ variable "slack_config_<teamn_name>" {
 All alerts are routed using the `severity` label. Provide the development team the severity label created for each route (default is team_name),
 which will be used by the developemt team when creating custom application alerts. 
 
-#### kube-prometheus-custom-alerts<application_name>.yaml
+#### prometheus-custom-alerts-<application_name>.yaml
+
+Once the route configuration is complete by the Cloud Platform Team, the application team to use the 'severity' label value supplied and create a custom alert using the template below:
 
 ```yaml
 apiVersion: monitoring.coreos.com/v1
 kind: PrometheusRule
 metadata:
   creationTimestamp: null
+  namespace: <namespace>
   labels:
-    prometheus: kube-prometheus
+    prometheus: prometheus-operator
     role: alert-rules
-  name: kube-prometheus-custom-alerts-<application_name>
+  name: prometheus-custom-alerts-<application_name>
 spec:
   groups:
-  - name: application.rules
+  - name: application-rules
     rules:
     - alert: <alert_name>
       expr: <alert_query>
       for: <check_time_length>
       labels:
         severity: <team_name>
       annotations:
-        description: <description_alert>
-        summary: <summary_alert>
+        Message: <alert_message> 
+        Runbook URL: <http://my-support-docs>
```

Comment:
Does `Runbook URL` work with a space - does this need to be `runbook_url`

Reply:
changed to match new template annotations